From ce7b1eff25d01ba733681957a38690ba4cd99ea5 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sun, 14 Feb 2016 14:43:52 -0500
Subject: [PATCH 3/7] fix enable_mt configure logic

In the default case, the build_mt flag is not set.  This leads to an
error at configure time like:
./configure: line 26866: test: =: unary operator expected

Change to use the variable name autoconf creates for us and quote it
so we never get weird expansion errors.
---
 configure.ac | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index c68bd44..73a5e2b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -60,10 +60,9 @@ CPIO_SYSDEP
 CPIO_PACKED_STRUCTS
 
 AC_ARG_ENABLE(mt,
- AC_HELP_STRING([--enable-mt], [Enable building of mt program]),
- [build_mt=$enableval])
+ AC_HELP_STRING([--enable-mt], [Enable building of mt program]))
 
-AM_CONDITIONAL([CPIO_MT_COND], [test $build_mt = yes])
+AM_CONDITIONAL([CPIO_MT_COND], [test "$enable_mt" = yes])
 
 AC_CHECK_HEADERS([unistd.h stdlib.h string.h fcntl.h pwd.h grp.h sys/io/trioctl.h utmp.h getopt.h locale.h libintl.h sys/wait.h utime.h locale.h process.h sys/ioctl.h])
 
-- 
2.20.1

